cpu/esp32/Makefile.include

```diff
@@ -151,6 +152,19 @@ else ifeq (120m,$(FLASH_FREQ))
   CFLAGS += -DCONFIG_ESPTOOLPY_FLASHFREQ_120M
 endif
 
+#extend CFLAGS by the corresponding FLASH_SIZE
+ifeq (1,$(FLASH_SIZE))
+  CFLAGS += -DCONFIG_ESPTOOLPY_FLASHSIZE_1MB
```

Comment:
Would it make sense to use 
```suggestion
  CFLAGS += -DCONFIG_ESPTOOLPY_FLASHSIZE_$(FLASH_SIZE)MB
```
instead or is there only a fixed set of `CONFIG_` defines anyway?

Reply:
These are the values declared by ESP-IDF's Kconfig.